app/src/org/commcare/android/javarosa/PollSensorController.java

```diff
@@ -112,27 +111,43 @@ public void onLocationChanged(Location location) {
         }
     }
 
-    @Override
-    public void onProviderDisabled(String provider) {
+    private void sendBroadcast(Context context) {
+        Intent noGPSIntent = new Intent(GeoUtils.ACTION_CHECK_GPS_ENABLED);
```

Comment:
it seems like this broadcase is getting used to convey any kind of location errors, I would suggest 2 naming changes based on that - 
`sendBroadcast` -> `broadcastLocationError` 
`ACTION_CHECK_GPS_ENABLED` -> `ACTION_LOCATION_ERROR`